src/components/Toolbar.vue

```diff
@@ -11,7 +11,7 @@
                 >
             </div>
         </transition>
-        <div class="input-icons">
+        <div class="input-icons" v-if="showIcon">
```

Comment:
@charlesstrange2 Should we use `v-show` instead? From the docs: https://vuejs.org/v2/guide/conditional.html#v-if-vs-v-show
```
Generally speaking, v-if has higher toggle costs while v-show has higher initial render costs. So prefer v-show if you need to toggle something very often, and prefer v-if if the condition is unlikely to change at runtime.
```

Reply:
I'd keep v-if.
v-show hides with css (iirc), and i think it's better if this never gets rendered to dom